client/app/src/Controller/Admin/OrganisationController.php

```diff
@@ -81,12 +87,15 @@ public function viewAction(Request $request, int $id)
             'currentTab' => $tab,
             'tabData' => $tabData,
             'count' => $result['count'],
+            'clientCount' => $clientCount,
```

Comment:
Thinking about this some more, should the client count be the count of the results anyway? Presumably `$clientCount` contains the count of all the clients, but that might be misleading if the results have been filtered.

Reply:
The client count currently includes archived and active clients whereas the paginated list includes all clients (active, discharged and archived) which confuses admin users as the counts don't match.
Phil has confirmed that OPG wants the client count to include all clients to match the number of clients listed in the paginated list. 
I noted a content suggestion in the ticket to separate out the counts like this:
- ‘Active clients: xxx’ 
- ‘Archived clients: xxx’ - which is conditionally displayed if an organisation has archived clients
Or I could just include a note that the client count includes discharged clients.

I'll check with Phil...

Reply:
Actually, is `$result['count']` just wrong? I would expect it to be the count of results, but it sounds like it's the count of all the clients? There's definitely a discrepancy somewhere which we need to iron out.

Reply:
The $result['count'] returns the total count of results depending on which tab you select. Yes, for clients it returns all clients (active, archived and discharged).